docs/Contributing/Audit-logs.md

```diff
@@ -1277,10 +1275,10 @@ Generated when a software installer is updated in Fleet.
 This activity contains the following fields:
 - "software_title": Name of the software.
 - "software_package": Filename of the installer as of this update (including if unchanged).
-- "software_title_id": ID of the edited software title.
 - "team_name": Name of the team on which this software was updated. `null` if it was updated on no team.
 - "team_id": The ID of the team on which this software was updated. `null` if it was updated on no team.
 - "self_service": Whether the software is available for installation by the end user.
+- "software_title_id": ID of the added software title.
```

Comment:
@rachaelshaw CI complained until I ran `make generate-doc` and auto-generated these changes. Looks like related to someone else's change that went though without doing this.

Reply:
We've fixed this on main already (today). Worth merging `main`.